index.html

```diff
@@ -218,6 +237,400 @@ <h2>
+          {{URLPatternInput}}s that define the [=within home tab scope|scope of
+          the home tab=] relative to the [=manifest URL=].
+        </p>
+        <aside class="note">
+          Each scope pattern is either a string which looks like a URL but has
+          "<code>*</code>" as a wildcard character, or a dictionary containing
+          separate URL components with "<code>*</code>" wildcards. See
+          [[urlpattern]] for more information.
+        </aside>
+        <p>
+          An application <dfn>has a home tab</dfn> if the [=Document/processed
+          manifest=] includes a non-null [=home_tab=] member of the
+          [=tab_strip=] member.
+        </p>
+        <p>
+          The <dfn>home tab context</dfn> is an optional [=application
+          context=] that has special properties compared to other application
+          contexts. If the application [=has a home tab=], every application
+          window SHOULD feature a [=home tab context=]. If not, then the
+          application windows SHOULD NOT have a [=home tab context=].
+        </p>
+        <p>
+          How the [=home tab context=] is presented is at the discretion of the
+          user agent, but it SHOULD have a different appearance to normal tabs.
+        </p>
+        <p>
+          A [=URL=] |url:URL| is said to be <dfn>within home tab scope</dfn> if
+          and only if:
+        </p>
+        <ul>
+          <li>the application [=has a home tab=], and
+          </li>
+          <li>|url| is [=manifest/within scope=] of the manifest, and
+          </li>
+          <li>at least one of the following:
+            <ul>
+              <li>|url| [=URL/equals=] the [=start URL=], with
+              [=URL/equals/exclude fragments=] set to true, or
+              </li>
+              <li>Applying [=URL pattern/match=] given any element of the
+              [=scope_patterns=] member of the [=home_tab=] member of the
+              [=tab_strip=] member of the [=Document/processed manifest=] and
+              |url| returns a {{URLPatternResult}}.
+              </li>
+            </ul>
+          </li>
+        </ul>
+        <p>
+          A URL is is <dfn>outside of home tab scope</dfn> if it is not
+          [=within home tab scope=].
+        </p>
+        <aside class="note">
+          <p>
+            The "home tab scope" (i.e., the set of [=URLs=] which are [=within
+            home tab scope=]) is the set of URLs to which the [=home tab
+            context=] can be navigated . All navigations within the application
+            to a URL [=within home tab scope=] will be performed in the [=home
+            tab context=], and all navigations to a URL [=outside of home tab
+            scope=] will be performed in a different [=application context=]
+            (i.e., another tab), creating one if necessary. The application's
+            [=start URL=] is always part of the home tab scope.
```

Comment:
Nit (and probably doesn't matter), but here you say the start URL is always in the home tab scope but then below you give a case where it isn't (because the scope is the empty set).

Reply:
"and probably doesn't matter" --> it's a web standard, everything matters! :joy: